sonar/modules/deposits/jsonschemas/deposits/deposit-v1.0.0_src.json

```diff
@@ -0,0 +1,485 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://sonar.ch/schemas/deposits/deposit-v1.0.0.json",
+  "title": "SONAR deposit v1.0.0",
+  "type": "object",
+  "required": [
+    "pid"
+  ],
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "default": "https://sonar.ch/schemas/deposits/deposit-v1.0.0.json"
+    },
+    "pid": {
+      "title": "Identifier",
+      "type": "string",
+      "minLength": 1
+    },
+    "_bucket": {
+      "title": "Bucket UUID",
+      "type": "string",
+      "minLength": 1
+    },
+    "_files": {
+      "title": "Files",
+      "description": "List of files attached to the record.",
+      "type": "array",
+      "items": {
+        "title": "File item",
+        "description": "Describes the information of a single file in the record.",
+        "additionalProperties": false,
+        "properties": {
+          "bucket": {
+            "title": "Bucket UUID",
+            "type": "string",
+            "minLength": 1
+          },
+          "file_id": {
+            "title": "File UUID",
+            "type": "string",
+            "minLength": 1
+          },
+          "version_id": {
+            "title": "Version UUID",
+            "type": "string",
+            "minLength": 1
+          },
+          "key": {
+            "title": "Key",
+            "type": "string",
+            "minLength": 1
+          },
+          "checksum": {
+            "title": "Checksum",
+            "description": "MD5 checksum of the file.",
+            "type": "string",
+            "minLength": 1
+          },
+          "size": {
+            "title": "Size",
+            "description": "Size of the file in bytes.",
+            "type": "integer"
+          },
+          "label": {
+            "title": "Label",
+            "type": "string",
+            "minLength": 1
+          },
+          "category": {
+            "title": "Type",
+            "type": "string",
+            "enum": [
+              "main",
+              "additional"
+            ],
+            "default": "main",
+            "minLength": 1
+          },
+          "type": {
+            "title": "Type",
+            "type": "string",
+            "enum": [
+              "file",
+              "fulltext",
+              "thumbnail"
+            ],
+            "default": "file",
+            "minLength": 1
+          },
+          "embargo": {
+            "title": "Embargo",
+            "type": "boolean",
+            "default": false
+          },
+          "embargoDate": {
+            "title": "Embargo date",
+            "type": [ "string", "null" ],
+            "format": "date"
+          },
```

Comment:
Strange for my point of view to have a key with `null` value. It seems better in this case to don't put the JSON key. And as this field seems not required (cfr L106), it's compatible with. In addition you could easily add a pattern to check the embargo date.